### ISSUE TITLE
The term "Affiliation" is used in several places but not strictly defined

### ISSUE THREAD

**Issue by cwilso** [w3c/w3process#176]
Although a "change of affiliation" is a triggering event in several cases, it is not strictly defined what an affiliation is.  Is it employment?  Contract?  Simple statement of "yes this individual speaks for us"?

May be related to issue #9 .

**Comment by Kometo2** [comment]
Well, i get endpoint

**Comment by chaals** [comment]
(This seems tangentially related to #9 but pretty closely linked to #175)

I think the key reference is [Section 3.1.2 Member Representatives](https://www.w3.org/2018/Process-20180201/#member-rep)

**If** we agree that "nominated as a representative by a member" is what affiliation refers to - my untested assumption - moderated by the requirements to notify the team of [Conflicts of Interest](https://www.w3.org/2018/Process-20180201/#coi) and the [Related Member provisions](https://www.w3.org/2018/Process-20180201/#MemberRelated) - perhaps the answer is to link the "change of affiliation" in the [section on triggering elections](https://www.w3.org/2018/Process-20180201/#AB-TAG-vacated) to section 3.1.2?

I note that the term "affiliation" is only used in the document in the sections mentioned above.

**Comment by nrooney** [comment]
Er, I am also going to link #179. Seems like we have a problem here (the only reasonable half-definition is in the [Conflict of Interest Policy](https://w3c.github.io/w3process/#coi)). Let me take a stab at it:

`disclaimer: this is strict. I added everything I could think of. Please feel free to add or suggest dialling it back!`

## Affiliation
Individuals participating within W3C will often have an "affiliation" to one or more organisations (@todo define organisations). "Affiliation" to an organisation will often be through employment (the individual is an "employer" of the organisation) although individuals MAY be "affiliated" through other means. The circumstances in which an individual is "affiliated" with an organisation are described below:

* The individual is an employer of an organisation whose activity is relevant to W3C
* The individual is being paid for consultancy work by an organisation whose activity is relevant to W3C
* The individual's participation in W3C is being supported financially by an organisation whose activity is relevant to W3C (travel, accommodation and associated travel costs)
* The individual is being contracted for work by an organisation whose activity is relevant to W3C
* The individual holds a seat on a board or leadership level group within the organisation whose activity is relevant to W3C
* The individual holds a seat on an advisory group within an organisation whose activity is relevant to W3C
* The individual holds and elected position on a board or group within an organisation whose activity is relevant to W3C
* The individual is a chair or holds a leadership position in a group within an organisation whose activity is relevant to W3C
* The individual is the author of a number of standards or specifications from an organisation whose activity is relevant to W3C
* The individual holds significant stocks, stock options, or other form of corporate equity in an organisation whose activity is relevant to W3C.

An individual MUST declare the organisation and the type of affiliation for each statement above which fits their situation. An individual MAY have multiple affiliations, and in which case, multiple of the above statements MAY be true. The individual MUST declare every true statement.

**Comment by cwilso** [comment]
I'll note a few things -
1) 'If we agree that "nominated as a representative by a member" is what affiliation refers to' - I do not, as that is a one-time event, while affiliation is a state.  For example, Microsoft nominated me a representative, and never retracted that nomination - but I am no longer affiliated with them.  A different event occurred - that of my departure from Microsoft - that ended that affiliation. That trigger - whatever changes an affiliation - is what needs to be defined.  I contend that termination of employment changes an affiliation.

2) @nrooney that's a long list, and likely not entirely tenable (e.g. I'm sure I own stock in multiple companies whose activities are relevant to the W3C, but I'm not sure we want to get into that level of disclosure for any of the places affiliation needs to be defined.

**Comment by dwsinger** [comment]
I think you (Natasha) might be trying to define clearly identified areas of doubt and uncertainty; affiliation/influence, and so on. Affiliation, for me, has a hard edge; it's a formal question of employment, contract, or other agreement. Influence ranges from the pernicious to the unremarkable.

**Comment by michaelchampion** [comment]
I like the first 4 bullets in @nrooney's list, that's what I think of as "affiliation" in the W3C context. 

The points about "affiliation" with other organizations that are relevant to W3C are worth discussing. I think that's information that would be useful to the AC in choosing AB and TAG representatives, but I'm not sure that a change in one of those roles at another organization should be considered a "change of affiliation".  

I'd definitely draw the line before "The individual holds significant stocks, stock options, or other form of corporate equity in an organisation whose activity is relevant to W3C." That would be impossible (or at least extremely intrusive) to verify and would mean that most people involved with W3C would "change affiliation" frequently.

**Comment by chaals** [comment]
@cwilso 

>  Microsoft nominated me a representative, and never retracted that nomination - but I am no longer affiliated with them. A different event occurred - that of my departure from Microsoft - that ended that affiliation. That trigger - whatever changes an affiliation - is what needs to be defined. I contend that termination of employment changes an affiliation.

Actually your affiliation is a datum recorded in W3C's systems tracking who is in what Working Group, and Microsoft almost certainly changed your affiliation you when you left. As a chair I get that sort of notification pretty regularly.

See also #179

**Comment by caribouW3** [comment]
In most parts where W3C systems use users' Affiliation, it is described as "significant employment relationship with". For WGs, it is definitely something like "will speak for" and, except for IEs, implies the organization's commitment to the Patent Policy. However, in CGs, people affiliated with a non-W3C-member organization may have the opportunity to speak for themselves ("join as individual"), as defined by the CGs rules.
It is not allowed to have multiple affiliations in a W3C account (technically speaking, as of today), but some people who represent different organizations in different groups (e.g. consulting for more than 1 member org) have several W3C accounts to make it clear who they speak for in a given context.

**Comment by cwilso** [comment]
@chaals Actually, your supposition is incorrect - Microsoft didn't change my affiliation when I left, and I'd expect several other large companies don't track and change this automatically.  I discovered, when I joined the Media and TV IG six months in to my time at Google, that my affiliation was still set to Microsoft.

**Comment by dwsinger** [comment]
Yup, there's no obvious way for the W3C to know. I am hooked into the Apple HR process and get termination reports quarterly, and do it all manually (for those I remember doing standards work...)

**Comment by chaals** [comment]
@cwilso colour me surprised. I know it is possible for this to occur, and like @dwsinger I spent a dozen years in two companies tracking employment changes weekly for this reason. 

(Still, it's an endorsement of your integrity that Microsoft were happy to trust you to represent them fairly even though you were no longer an employee.)

There are various cases where organisations big and small **do** nominate people to represent them who are not employees. The reasons why such a representative need not be an employee are essentially a matter between the member and the representative IMHO, so long as the disclosure requirements in the Process are met.

However, you highlight the fact that AC representatives who don't track what their organisation is doing (or organisations who don't track what their AC rep does) may end up in a situation they don't like much.

We can optimise either way. My own preference is to keep it simple for engaged members / AC reps. YMMV...

**Comment by dwsinger** [comment]
3.1.1 uses "affiliation" and therefore this entangles conflict of interest

**Comment by dwsinger** [comment]
perhaps there is a question here of whether the organization that nominated no longer sustains that nomination i.e. they declare that the individual no longer represents them?

**Comment by dwsinger** [comment]
we might need to edit the last sentence of 3.1 preceding 3.1.1 to explicitly include the COI policy

**Comment by frivoal** [comment]
(this comment was previously posed in #186, but it is better here, so moving it)

Some cases of change of affiliation are easy, some are not:

Easy case:
1. A person used to be an employee / director of a member company, now is an employee / director of a different member company

Various less obvious cases:

2. A person used to be self-employed, has now incorporated.
3. A person used to be an IE, now (without change of employment or incorporation), has become a paying member…
    1. … because they just joined as an individual
    2. … because the company they own/control just joined
    3. … because the company that employs them just joined
4. 2 + 3.ii at the same time
5. A person used to be an employee / director of a member company, now is a contractor of the same member company
6. A person used to be a contractor for several member companies, now is an employee / director of one of them
   1. part time, and retains their previous contracting clients
   2. full time, and no longer works for their previous contracting clients.
7. A person used to be an employee or contractor of a member company, now no longer receives any financial compensation from that member company, but the company is still willing to let the person participate in their name
8. A person who used to be employee / director in one member company continues to be, but now is additionally director / employee in one additional company…
   1. … which is a member company
   2. … which is a not member company
9. A Invited Expert used to be self employed, and now is the employee of an company which is not a member
10. A Invited Expert used to be and employee / director of a company which is not a member, and now is the employee / director of an other company which is also not a member

(Plus, any of the above in the opposite direction.)

While I don't think all of the above have necessarily happened to AB/TAG members yet, I can think of concrete examples of individuals active in W3C for pretty much all of the above (hell, I've probably been through half of these myself…)

The above laundry list is certainly not something we should incorporate into the process directly, but some high level principle by which we could judge which case falls on which side would be welcome.

----
My take is that:

2, 3.i, 3.ii, 4, 5, 7, 8.ii, 9, 10 don't seem useful to me to classify as changes in affiliation

3.iii, 6.ii are probably worth counting as changes in affiliation

6.i, 8.i I don't know.

----
I think the underlying principle behind that classification is that, as far as their w3c activity is concerned, if a person switches between being someone's agent to someone else's agent, or from being someone's agent to representing themselves, it is a change of affiliation, and otherwise, it is not, and that you can only be someone's agent in w3c if that someone is a member (otherwise you're representing yourself).

**Comment by dwsinger** [comment]
Perhaps we can state that a change of affiliation is either (a) a change of employment or (b) the originally nominating member organization withdraws their support, i.e. would not re-nominate now?

Other conflicts, affiliations, interests, I think can be the part of reasonable disclosure.

**Comment by dwsinger** [comment]
@frivoal wow, thanks. The slight complexity is what about the nominating company? If they "withdraw their nomination" (i.e. support) of a member, does that have any effect?  Half of me says no, once someone is *elected* they are on the TAG/AB by virtue of election, and their nomination status was only material to the election. Half of me says yes, it matters, especially if the individual has no other link to the W3C (i.e. they were nominated as an external individual).

**Comment by frivoal** [comment]
I think we should not make it a possibility for companies to withdraw AB/TAG nominations. Of course the relation they have (employment, contracting, whatever) they have with the individual may evolve, and we need decide what these various changes mean, but having the ability to just renege on a nomination without any other change in the relationship with the individual gives a very low-cost way to pressure the individual into defending positions that the company wants them to, even though they are supposed to participate as themselves, not as a representative of the company.

Certainly, in most cases, the nominating company is the employer, and they already have numerous means to apply pressure should they want to. But no need to add more.

**Comment by jeffjaffe** [comment]
I like Natasha's point above that we need to link our entire treatment of "Affiliation" with our "COI policy" mor generally.

**Comment by dwsinger** [comment]
I support the idea of a sentence “for the avoidance of doubt, the following are always considered a change of affiliation” and a section “other changes of affiliation fall under the disclosure policy of conflicts of interest”.

**Comment by cwilso** [comment]
Again:  NOMINATING someone for an AB position is a single-point-in-time
action, and it is NOT the same as designating your AC rep anyway -
WITHDRAWING a nomination can only happen before the election.  Google might
have decided at any point in the last six months that having me on the AB
was bad for it, but they couldn't withdraw my nomination and force me to
leave the board.  I've served at the pleasure of the entire AC, not my
employer.  I'm not "serving on the AB in Google's name" - I was elected (or
not :) by the AC.  Affiliation is a state, and has to do with motivations,
which can sometimes be hidden.

HOWEVER, where my major sources of income come from may make a difference
in my motivations, and who employs a given AB member certainly makes a
difference in my (as a proxy for an AC member) perception of their
motives.  Thus, I would say quite strongly:
1,2,3,4 are not an affiliation change.
5,6,7,8 ARE an affiliation change (5 and 8 are iffy, but better safe than
sorry).
9,10 absolutely are an affiliation change.

**Comment by chaals** [comment]
[@frivoal's opposition to allowing withdrawal of affiliation](https://github.com/w3c/w3process/issues/176#issuecomment-396874571) makes a lot of sense. The one case where it is problematic is if a Member has nominated an employee, and a year later wishes to withdraw that employee to nominate another. (This has actually occurred, as a result of corporate acquisition), as an alternative to nominating another employee, and if that person is elected the first resigns, typically triggering a special election.

**Comment by chaals** [comment]
Note that if we have an executive rather than Advisory Board, or give the TAG normative decision power over other groups' work, this and related questions are likely to be more pointed than at present.

**Comment by dwsinger** [comment]
@frivoal I think I agree. 

The only reason I raised it is this: at the moment, it's possible for a company to nominate a non-W3C member and for them to be elected; then the person becomes difficult to work with, problematic, disruptive, and the company and that person fall out and cease to have a relationship. At that point we have a problematic person on the AB or TAG with no 'link' to the W3C. We have no 'recall' (not that I would support being able to 'recall' TAG/AB members). We don't have a member who we can work with to deal with the disruption. Unless they violate some rule we are stuck with them without any 'hook' to handle the situation.

**Comment by frivoal** [comment]
https://github.com/w3c/w3process/issues/163 is related, we should look into it at the same time as this issue.

**Comment by jeffjaffe** [comment]
Mike, these are some excellent thoughts.

I'm unsure whether we would want to have such a lengthy discussion in the process document (which we are trying to make shorter).  Perhaps we can give summary guidance:

Affiliation: Consider organizations of relevance that you belong to - (principally) employer, but also business partners, other standards organizations, position within your employer, contractual relationships, etc.

Change in Affiliation.  If any of the affiliations change, in a way that there is a scent that your point-of-view might change as a result, then it is a Change of Affiliation.

Then we could give a non-normative pointer to a broad discussion of use cases.

**Comment by michaelchampion** [comment]
@jeffjaffe I agree, that's about the right level of detail for the Process Doc.  We want to encourage people to be transparent (my word for the day, I guess) about affiliations but for W3M to force early election campaigns only when it is clear that a reasonable person might question whether the assumptions the AC made during the original campaign are still warranted.

I assume you meant "there is a SENSE that your point-of-view might change …" but "scent" is quite apt as well. ;-)

**Comment by frivoal** [comment]
I think clarity here is important. Since the effect of affiliation changes is quite dramatic on these elected positions, fuzzy rules may give the feeling that they can be selectively enforced. It less Machiavellian terms, it may also make the elected individuals question whether they should make certain career moves.

We cannot be fully exhaustive or fully specific, as this is a complicated situation, but some general guidance would be welcome.

> If any of the affiliations change, in a way that there is a scent that your point-of-view might change as a result, then it is a Change of Affiliation.

That's too strong, especially when you list contractual relationships as part of affiliation. This could be interpreted that Mozilla's representative needed to step down when Firefox's search integration switched from Google to Yahoo, which is overkill.

I'll worry about clean phrasing later, but to me, the essence is whether there is a change in terms of principal-agent relationship. If you were in you main job the agent of someone, and now you're the agent of someone else, or if you were your own principal and now you're someone's agent, or vice-versa, there's a change in affiliation.

**Comment by dwsinger** [comment]
I think we need to distinguish disclosures of potential conflicts of interest, from changes of affiliation. Changes of affiliation trigger automatic actions, so we probably want to be careful when they trigger. We currently do not have a mechanism for considering a conflict of interest sufficiently serious to trigger action; should we? I think the change of affiliation rule should be, to the greatest extent possible, clearly defined such that little or no judgment is needed as to whether a change has occurred. I like Florian's direction; maybe when running for the TAG or AB you should be required to disclose your primary affiliation/agency-relationship, and if that changes (such that if you were today you'd have to say something else) your affiliation has changed.

**Comment by frivoal** [comment]
> maybe when running for the TAG or AB you should be required to disclose your primary affiliation

Isn't that the case already? That's how I read (and filled) this form:
https://www.w3.org/2002/09/wbs/1/abnominee2018/

**Comment by dwsinger** [comment]
No. The form asks what your 'other affiliations' are (for conflict of interest), whether you are an employee of a member, and whether you'll get financial support, but it doesn't ask "What is your primary affiliation?" (for most people, this will be their employer)

**Comment by jeffjaffe** [comment]
@dwsinger Your distinction between potential CoI and changes of affiliation is compelling.  Since the AB is elected by the membership, I would be interested if we could somehow get a broader viewpoint from them, as to what level changes of affiliation and CoI should require resignation from the AB and TAG.

**Comment by michaelchampion** [comment]
> what level changes of affiliation and CoI should require resignation from the AB and TAG.
> 

"Resignation" is not what would be triggered a change of affiliation, it's a requirement to run in the next election, right?

**Comment by dwsinger** [comment]
Yes, to be clear:
Change of affiliation causes the seat to be declared vacant at the next election.
Conflicts of interest are provided to the community for their information, and have no automatic effect.

We actually have no 'recall' or 'ask for resignation' option even for the most serious conflicts of interest.

So, my strawman is very simple:
* Candidates must declare/identify their primary affiliation on the nomination form.
* If the status of a sitting member changes such that they no longer have that primary affiliation, that is a change of affiliation (i.e. if they were to fill the form in anew, they'd have to say something materially different).

So, resigning/being layed off, joining a new company, are obvious changes of affiliation.
A rename of the company is not a material change; working for yourself and then incorporating are not material changes either.
If you were nominated by a member not as employee but as a consultant, supported by them, and they terminate the consultancy, that was your primary affiliation, and the termination changes that.

All the grey areas fall into COI, not change of affiliation, IMHO.

**Comment by jeffjaffe** [comment]
@dwsinger I have no strong personal opinion on the matter.  I was only pointing out that if we are "opening this box" in the process document, it would be worthwhile to ask the members whether a strong enough COI (or change to secondary affiliation) should also force someone to run in the next election (concurring with Mike's correction above).

The rational is that in some cases, some new COI might be a stronger change then a small change in affiliation (e.g. an IE who gets some contract from company X actually joins company X; or a person changes from one SemWeb startup to a different SemWeb startup).

**Comment by dwsinger** [comment]
If someone discloses a sufficiently serious conflict of interest, I wonder if we already have enough tools. (a) The Director or anyone else can suggest that remaining on the TAG/AB is no longer tenable or even (b) the Director can use "The Director may suspend or remove for cause a participant in any group (including the AB and TAG), where cause includes failure to meet the requirements of this process, the membership agreement, or applicable laws."

So perhaps the 'automatic' rule can be based on the simple case -- change of primary affiliation -- and everything else can be left to community and specifically Director judgment?

**Comment by jeffjaffe** [comment]
@dwsinger As I said above, if the membership only wants to make "automatic" based on change of primary affiliation, I have no strong objection to that.

However, I disagree with the expectation that anyone would tell someone to leave the AB or TAG, unless it was clearly specified in the process document that it is required.

**Comment by dwsinger** [comment]
@jeffjaffe OK, so the alternative is that we have a mechanism/policy for 'serious conflicts of interest'. This seems harder to define or manage. I can't see a 'recall election' mechanism that would work, and 'serious' is a judgment call. 

We could adjust the text I quoted above to add "serious conflicts of interest", i.e. "The Director may suspend or remove for cause a participant in any group (including the AB and TAG), where cause includes serious conflict of interest, or failure to meet the requirements of this process, the membership agreement, or applicable laws."

**Comment by michaelchampion** [comment]
Code of Conduct violations are the only grounds I can imagine for the Director removing someone from the AB or TAG.  That's already covered in the process document section 3.1.

I could imagine (Tantek proposed this on the CG call yesterday) adding the ability to have a recall election so the AC can judge whether some conduct is incompatible with continued service on the AB or TAG.  I'm not opposed but would be inclined to wait for an example where the Director failed to take appropriate action before investing time in crafting process language for this.  In a hypothetical new legal entity where there the Director is not the final arbiter of everything, recall by the AC might be the appropriate way to handle such situations, however.

**Comment by chaals** [comment]
@dwsinger wrote

> No. The form asks what your 'other affiliations' are (for conflict of interest), whether you are an employee of a member, and whether you'll get financial support, but it doesn't ask "What is your primary affiliation?"

That's probably because your affiliation is a piece of data maintained as part of your W3C account information, and meant to be kept up to date. If for example the nominating member leaves W3C you are automatically dropped from various groups, but where an AC rep nominates you as affiliated with them, and the relevant member remains a member, it is up to the AC rep to decide when to unaffiliate you, unless you decide to change your own affiliation.

**Comment by chaals** [comment]
The Conflict of Interest policy is pretty important IMHO.

But the process document says quite clearly that being on the AB or TAG means you are meant to represent the best interests of W3C and the Web, **NOT** your employer.

Too much focus on your employer beyond the CoI requirements seems to suggest that this requirement isn't meant to be serious. I think that would be an extremely unfortunate impression to give, especially to potential candidates and to those who support them either financially to participate or as voters.

It also seem to presuppose a particular view of employer-employee relationships that in my experience with W3C is often very different from reality.

**Comment by michaelchampion** [comment]
> But the process document says quite clearly that being on the AB or TAG means you are meant to represent the best interests of W3C and the Web, NOT your employer.
> 
> Too much focus on your employer beyond the CoI requirements seems to suggest that this requirement isn't meant to be serious

This is a case of  Reagan's favorite Russian proverb: Доверяй, но проверяй. At some level we **trust** and expect the TAG and AB to represent the best interests of the W3C, but it's important to **verify** their economic and social relationships.  For the foreseeable future we will be electing humans to the AB and TAG, and human cognition is well known to be biased by social and economic relationships, so it's not prudent to simply assume and assert that people will represent the Web rather than their own interests just because that's written in the process doc.

So having current information on who is paying AB and TAG members goes beyond ensuring there's no blatant conflict of interest, we need to understand where people sit in social and economic networks to assess where they really stand in discussions.  This is how humans work, for better or worse.

**Comment by cwilso** [comment]
I'm just coming back from a long vacation, but I want to state that clearly
just representing the best interest of the Web and the W3C on the AB or Tag
is NOT enough, as there is a clear prohibition between having multiple
members of those august bodies with the same employer.

Chaals, I respect that YOU believe "it is up to the AC rep to decide when
to unaffiliate you, unless you decide to change your own affiliation,"  but
I respectfully and strongly disagree.  When your employment changes (e.g.
you leave your position at an employer), I believe it is a clear change in
your affiliation.  If your CoI status (when hired by a Member with an
affiliate already on the AB/TAG) isn't affected by your former employer
saying "oh yeah, I still vouch for them", then I don't see why it would be
true for your affiliation status.  Your relationship has fundamentally
changed.

As Mike says, humans, economic and social relationship, trust but [give the
AC through its vote an opportunity to] verify.

**Comment by chaals** [comment]
TL;DR: I am not going to raise an objection attempting to block consensus on whatever the resolution of this is, noting that others have explained reasonable grounds for considering the issue differently. Nor do I support a change to defining affiliation formally as "a formal employment relationship" because I don't think any real problem it solves is even as large as those few I believe it creates.

Some rationale not mentioned in earlier comments [1](https://github.com/w3c/w3process/issues/176#issuecomment-378321149) [2](https://github.com/w3c/w3process/issues/176#issuecomment-378321149) [3](https://github.com/w3c/w3process/issues/176#issuecomment-397223651):

The current situation as I interpret it is that an AC rep says "we think this person will do a good job in Working Group X", or "as an AB member", or whatever, and nominates them.

People are elected as individuals. Individuals with sufficient profile to be elected are **very often** independent thinkers prepared to express their own opinions rather than some "company line". In practice I believe that helps a number of people get elected.

@michaelchampion 

> ...having current information on who is paying AB and TAG members goes beyond ensuring there's no blatant conflict of interest, we need to understand where people sit in social and economic networks...

I agree in principle. But in practice I think the proxy of "employer" is pretty poor, and there is a risk of checking comforting but misleading information, just because it is apparently measurable.

Large companies in our space rarely behave as a single hive mind, and often have a wide range of disparate views even within a relatively small department. Many academic institutions and news providers, among others, still prize such freedom very highly. 

Earlier, Mike also wrote:

> elected members should be scrupulous about telling the membership how their participation is funded if that doesn't come from formal employment.

This sounds sensible, until you get to the details. Do we ask an employee who is given some time and support for work, and covers some costs themselves, how they are funded? Or someone who is reapplying for their job as part of a corporate restructure, or someone who is occasionally given support from a third party to cover a meeting their normal employment would not cover?

That seems to reinforce the already strong structural bias toward wealthy corporations as the source of AB and TAG members noted in #189, and I don't see that as helpful.

**Comment by chaals** [comment]
(Sorry, I should have written this earlier, because others may well be following along)

**DISCLAIMER**
In case anyone reading this is unaware, the question applied to me recently. I explained my reasons for considering I had not changed affiliation in the sense intended in the Process document, and offered to follow the guidance of the AB should they reach a different resolution.

**Comment by dwsinger** [comment]
@chaals is right, the W3C already maintains a formal primary affiliation for someone. So perhaps we need to say that when the nominees are announced, their primary affiliation is also announced, and if it changes, then the clause kicks in. This does mean that if someone leaves a company but no-one bothers to inform the W3C formally, we may have an untruth; but if someone is worried about their status, I don't see any reason why someone else can't inform the W3C.

**Comment by cwilso** [comment]
That affiliation is not clear, then, because it would seem that it used for the participation constraints in 2.5.1 (although I'd point out that after going on about "with the expectation that they will use their best judgment to find the best solutions for the Web, not just for any particular network, technology, vendor, or user," saying "A Member organization is permitted at most one participant" is an odd way of saying it - as if they control that person and that person is "theirs".

Would it have been okay for Microsoft to continue my affiliation after I had left their employ, if they believed I was still "affiliated" with them?  How about if they considered me "their" participant on the TAG, say, because Google already had one?

Affiliation should be a definite test, and used for both of these clauses.

As to your point, @chaals, about asking how non-large-corporate participants are funded, that question can be asked and answered when they are nominated and elected or appointed.  If the status of it radically changes, I'd expect them to be honest, of course - but I see far few concerns there than from a change of overall employment (and employer willing to fund that participation).

As for "reapplying for their job as part of a corporate restructure" - if their employment continues, I don't see the issue.  If they move to a different division, I don't see the issue (unless it's a different subsidiary that is a separate W3C Member).  If someone who is occasionally given support from a third party to cover a meeting their normal employment would not cover, I'd hope they would disclose that, but again - I probably voted for them (or not) with that understanding.

I disagree with your statement "[this] seems to reinforce the already strong structural bias toward wealthy corporations as the source of AB and TAG members".  To be honest, as a current and past employee of two different organizations you'd probably put in the category of "wealthy corporations," I can tell you that it means that I'm already painted with the stigma of being "[wealthy corporation]'s shill" whether I like it or not.  What causes a bias toward those other organizations providing candidates is that they can afford it (and, of course, the broader web ecosystem success is important to them, but that's true of other orgs too).

**Comment by dwsinger** [comment]
Trying again!

When a candidate filing is made, the candidate's primary affiliation as recorded in the W3C database is reported to the electorate.
If the status of a sitting member changes such that they no longer have that primary affiliation, that is a change of affiliation (i.e. the database has, or should have, changed).
So, resigning/being layed off, joining a new company, are obvious changes of affiliation.
A rename of the company is not a material change; working for yourself and then incorporating are not material changes either.
If you were nominated by a member not as employee but as a consultant, supported by them, and they terminate the consultancy, that was your primary affiliation, and the termination changes that.

In addition, we should add "serious conflicts of interest" as being another cause that the Director may use to dismiss someone from the TAG or AB.

any closer?

**Comment by jeffjaffe** [comment]
@dwsinger We recently had a situation when someone left a company, but argued that from the point of view of W3C, was still affiliated with that company.  I believe that this person would argue that the second paragraph of your proposal could support their position, whereas the third paragraph would not support their pov.  So it is not clear to me that your proposal disambiguates that case.

**Comment by chaals** [comment]
I am the person referred to in the previous comment, and I would not argue as suggested.

I would point out that in @dwsinger's proposal, if you no longer do paid work for the organisation, but the organisation once again confirms that you represent them (as happened), it is not clear how the change in affiiliation from company X to company X should be handled. While I am pretty sure it can be included, I am not sure that focusing too closely on the one recent example is the way to make the process clearer.

What happens if someone is occasionally doing paid work for, and is continually representing a member? E.g. a contract of some months duration every other year or so

What happens if you are employed by some organisation but are nominated as representing a different member, and subsequently change employment to that member? (Presumably, nothing, but again this isn't clear).

**Comment by dwsinger** [comment]
I see all these grey areas, but this is the best I can do for a bright-line rule. I welcome improvements. I think anything mandatory "if X happens then the affiliation has changed" needs to be outside a judgment call. I welcome suggested improvements; the discussion is nice, but we need to get soon to actual text...

**Comment by michaelchampion** [comment]
So the "bright-line rule" is that "affiliation" == "employment"?  And the Director can make a judgment in gray areas involving different forms of self-employment and contractual relationships with another entity?

**Comment by dwsinger** [comment]
The bright line is who your recorded affiliation is -- typically employer, or contracting entity.  And my additional proposal is that the we add "serious conflicts of interest" as a case where the Director can exercise judgment. It's not ideal, but is there better to be had?

**Comment by nrooney** [comment]
So a individual can be affiliated with more than one organisation, correct? As the process document is written it almost sounds like it's a one-to-one relationship, maybe we should make some small adjustments to suggest it could be one-to-many?

**Comment by frivoal** [comment]
> So a individual can be affiliated with more than one organisation, correct?

Depends what you mean.

If you mean somebody who is for instance employed by one member, and on the board of another one, and nominated to various W3C groups by each, then yes, that seem fair to say they have multiple affiliations.

But as a freelance consultant, I don't want my affiliation to be all my contracts, and to be considered to have changed anytime I break or make a new one. Not unless (just an example) the affiliation of everybody who works for Mozilla is considered to have changed when Mozilla changed their search deal from Google to Yahoo. Which would be silly. I consider my affiliation to be myself. Disclosing my contracts for the sake of checking on conflicts of interests is OK (though we don't require the same of member companies), but that's not affiliation.

**Comment by dwsinger** [comment]
I think if you are linked to multiple companies (e.g. as a contractor) your primary affiliation is the contracting company, i.e. yourself, and the contracting relationships should be disclosed appropriately according to the Conflict of Interest policy. At least, that would seem best practice.

If you get nominated by another company you are contracting for, and the nominator declares themselves to be your primary relationship, then when the contract finishes, so does your primary affiliation, and we'd need to declare the seat vacant at the next election.

I lean backwards and forwards on whether the nomination form should ask for your primary affiliation, or whether we should pick it up from the database at the time of nomination. If we do the latter, I think the nomination form had better be upgraded to report it ("The primary affiliation of Fred Bloggs will be shown as Mad Magazines, Inc.").

**Comment by dwsinger** [comment]
@cwilso , I tend to agree with you. That's what I am trying to get to formalize. Help me with procedures and text?

So, maybe we need some informative text in the form? Something like this?

"For most people, your primary affiliation is your employer that is also a W3C member, and is your recorded affiliation in the W3C database. If you leave that employment, your affiliation ceases (unless replaced by a consulting or other formal relationship). In the case of contractors, your primary affiliation is probably your consulting company, with disclosure of your potential conflicts of interest in consulting relationships. If you declare your consulting agreement to be your affiliation, then a termination of that consultancy would also change (terminate) your affiliation." ??

**Comment by cwilso** [comment]
Sorry, I should have been more clear.  I largely agreed with your wording.  I would use the word "change" rather than "cease" and "terminate", but other than that, I think it's 👍.

**Comment by dwsinger** [comment]
@cwilso so, maybe we can craft text for the process and the form and so on.

"The nomination form MUST ask for the nominee's primary affiliation, and this will be reported on the ballot. For most nominees, the primary affiliation is their employer and will match their affiliation in the W3C database. For contractors and invited experts, this will normally be their contracting company or their invited expert status; in some cases (e.g. where a consultant is consulting for only one organization) this may be the organization for whom the nominee is consulting. A change of affiliation is defined such that this field would carry a different answer if the nominee were to be re-nominated. Other formal relationships (e.g. other contracts) should be disclosed as potential conflicts of interest."

The form would have a field for this; ideally pre-populated with the database affiliation record. And with the text associated:
"For most people, your primary affiliation is your employer that is also a W3C member, and is your recorded affiliation in the W3C database. If you leave that employment, your affiliation changes (unless replaced by a consulting or other formal relationship). In the case of contractors, your primary affiliation is probably your consulting company, with disclosure of your potential conflicts of interest in consulting relationships. If you declare your consulting agreement to be your affiliation, then a termination of that consultancy would also change (terminate) your affiliation." 

Is this getting close?

**Comment by cwilso** [comment]
LGTM.

**Comment by dwsinger** [comment]
change "your consulting agreement" to say the organization for whom you are consulting.

**Comment by nrooney** [comment]
note to self - add @dwsinger's text with his changed text in the comment immediately above, raise PR

**Comment by frivoal** [comment]
So, it seems we do not have clear agreement one whether that text is good. Recapping what I understand the issue to be:

* AB members and TAG members must stand at the next election if their affiliation changes. We do not know for sure what affiliation means, nor weather it means the same thing for AB or TAG. This is the problem we're trying to solve, because we have been faced with situations where the answer was non obvious, and where we could not agree.

* The proposed text defines affiliation as the same thing for AB and TAG, in a way that does not completely eliminate gray areas, but reduces them.

I feel this is a quite verbose piece of text that does not buy us that much, but which does bring some clarification.

Here are a few examples of concrete situations that would be good to disambiguate:

1. Someone is elected while being an employee of a company. During their mandate, they leave that company, but the company does not disclaim them

2. A self employed person incorporates

3. An independent consultant (whether self employed or incorporated) gains or loses a customer.

The proposed wording clarifies that neither 2 nor 3 are affiliation changes in general (although 3 can be if the consultant declared their sole customer as being their affiliation). It does not clarify 1.

I believe @LJWatson suggests that for the AB, we define change in affiliation to be a change in conflict of interest disclosure, and I think that this is far too sensitive.

To quote the Process about Conflicts of Interests:

> Individuals participating materially in W3C work must disclose significant relationships when those relationships might reasonably be perceived as creating a conflict of interest with the individual's role at W3C.

So changes in disclosures of conflicts of interest are not merely expected when there are actual problems, but for all things that *might be perceived as* potential problems.  Potential conflicts of interest change all the time.  Revoking someone's elected position on these grounds sounds improper.

**Comment by cwilso** [comment]
1.  should absolutely be a COA.  If someone who is elected JOINS a company during their mandate, this is a change of affiliation - it's irrelevant whether the company "disclaims" them or not.

I don't think 2 or 3 are important.  As to the point that things might be perceived as potential problems - yes, if I leave EFF and go to work for Facebook - or the opposite - that may change how people view me, whether my former affiliation "disclaims" me or not.

**Comment by michaelchampion** [comment]
On the call today, someone argued that since AB members are elected as individuals, a change in their employment is irrelevant.  I disagree pretty strongly.  As @cwilson notes, a change in employment can change perceptions even if a person tries very hard to make decisions on the basis of what is best for W3C or the web rather than one employer or another.

Another reason why employment is important even when people are chosen as individuals is that there is much evidence that humans UNCONSCIOUSLY make "biased" decisions that favor themselves or their social network, even if they try not to (or think they are not doing so).  See https://en.wikipedia.org/wiki/List_of_cognitive_biases for numerous examples.  

So we can continue to aspire to elect people to the AB who will act as individuals, but should also ensure the AC can re-assess their suitability if their primary employment affiliation changes in a potentially significant way. I agree that 2 and 3 in @frivoal 's list above are not likely to be significant.

**Comment by dwsinger** [comment]
@michaelchampion I agree with you; a change of primary affiliation is very relevant. For example, whether or not I am getting advice from my corporate lawyers is pretty significant in terms of my AB contribution.

@frivoal I agree, the major case is change of employer; but in the process, we try to be roughly complete.

So, trying to deal with the questions so far:

"The nomination form MUST ask for the nominee's primary affiliation, and this will be reported on the ballot. For most nominees, the primary affiliation is their employer and will match their affiliation in the W3C database. For contractors and invited experts, this will normally be their contracting company or their invited expert status; in some cases (e.g. where a consultant is consulting for only one organization) this may be the organization for whom the nominee is consulting. A change of affiliation is defined such that this field would carry a different answer if the nominee were to be re-nominated (therefore, terminating employment, or accepting new employment, are changes of affiliation). Other formal relationships (e.g. other contracts) should be disclosed as potential conflicts of interest."

The form would have a field for this; ideally pre-populated with the database affiliation record. And with the text associated:
"For most people, your primary affiliation is your employer that is also a W3C member, and is your recorded affiliation in the W3C database. If you leave that employment, your affiliation changes (unless replaced by a consulting or other formal relationship). In the case of contractors, your primary affiliation is probably your consulting company, with disclosure of your potential conflicts of interest in consulting relationships. If you declare that the company for whom you consult is your affiliation, then a termination of that consultancy would also change (terminate) your affiliation."

We also need the team to comment on the possibility of adding
a) the ability to declare primary affiliation on the nomination form
b) the ability to include the advice, above, with that field

**Comment by LJWatson** [comment]
@michaelchampion:
>On the call today, someone argued that since AB members are elected as individuals, a change in their employment is irrelevant. I disagree pretty strongly.>

Not quite. What I said was that in the Process it currently says that TAG/AB members are elected in their personal capacity, and the only time a change of affiliation is relevant is if it triggers the constraint of participation clause.

**Comment by frivoal** [comment]
@LJWatson I am confused. there's the following sentence in 2.5.3

> If a participant changes affiliation, but the participation constraints are met, that participant's seat becomes vacant at the the next regularly scheduled election for that group.

As far as I can tell, there are 2 cases where affiliation of TAG/AB members are relevant:
1) it changes and triggers the constraint of participation clause => the seat is immediately vacated
2) it changes and does not trigger the constraint of participation clause => the seat is vacated at the next election, which may cut the term from 2 years to 1 year, if it happened during the first year

I'm trying to understand if you're saying that 2 isn't actually a real thing (for some reason that escapes me), or that it shouldn't be a real thing (so we should delete the sentence above), or if you mean something else.

**Comment by frivoal** [comment]
@dwsinger notwithstanding Jeff's concerns about this being potentially too verbose compared to its importance, I find this definition to be useful. It significantly reduces uncertainties, and clarifies situations that previously were not clear. If you take the laundry list of sample situations I wrote in https://github.com/w3c/w3process/issues/176#issuecomment-396516561, the text provide clear answers or strong hints to most situations.

I don't find any of these conclusions objectionable, but interestingly, some of them did not match my initial intuition pre-definition, and to me that means it is indeed good to have a definition.

The only one in the list that I'm not sure how to answer based on the text above is point 8, and I think that's fine. There will always be corner cases.

TL;DR: +1

**Comment by dwsinger** [comment]
so, are we OK with this definition? If so, we're waiting for team comment on changing the form (possibly pre-populating the 'affiliation' field with the info from the database).

**Comment by jeffjaffe** [comment]
@dwsinger In parallel to the discussion about what should be in the process doc on affiliation, I'll ask the team about changing the form.  imho it does not appear to be a big deal.

**Comment by LJWatson** [comment]
"so, are we OK with this definition?"

Yes, I think so.

**Comment by jeffjaffe** [comment]
@dwsinger I checked with the Team.  No problem at all with making changes in the form.  Pre-populating a field in WBS, however, is not trivial - so they recommend that we simply add a field to list Affiliation.

**Comment by dwsinger** [comment]
looks like this can go to the editor for the process and the team (when the process is adopted) for the nomination form.

**Comment by frivoal** [comment]
Closed by #203
